internal/app/kdk/snapshot.go

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2018 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kdk
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+func Snapshot(ctx context.Context, dockerClient *client.Client, logger logrus.Entry) error {
+
+	snapshotName := Name + "-" + strconv.Itoa(int(time.Now().UnixNano()))
```

Comment:
Where does Name come from?  Is it a global?

Reply:
https://github.com/cisco-sso/kdk/pull/30/files#diff-c9bb13bf6d2baafd0e7014870515300fR35

Reply:
var scope is limited to `package kdk`  _i think_